### PR TITLE
Enable gpt_for_sequence_classification benchmark

### PR DESCRIPTION
**Pull request by FrostML**
### PR types

Others

### PR changes

Others

### Description

Enable gpt_for_sequence_classification benchmark.

### CODE REVIEW

**Review by mmglove — COMMENTED** [review]
LGTM

File: tests/test_tipc/configs/gpt_for_sequence_classification/train_infer_python.txt [hunk 1]
```diff
@@ -51,7 +51,7 @@ null:null
 null:null
 ===========================to_static_train_benchmark_params===========================
 to_static_train:--to_static
-===========================disable_train_benchmark==========================
+===========================train_benchmark_params==========================
 batch_size:2|8
```
Comment (mmglove) [inline tests/test_tipc/configs/gpt_for_sequence_classification/train_infer_python.txt]: 只需要添加一个batch_size

**Review by mmglove — COMMENTED** [review]
请贴上3次稳定性自测数据

**Comment by FrostML** [comment]
三次性能数据如下：


PaddleNLP_gpt_for_sequence_classification_bs8_fp16_DP_N1C1 | 90407.35023 | 91116.68744 | 89700.04913
-- | -- | -- | --
PaddleNLP_gpt_for_sequence_classification_bs8_fp16_DP_N1C1_d2sT | 88890.16669 | 87659.75095 | 88989.69092
PaddleNLP_gpt_for_sequence_classification_bs8_fp32_DP_N1C1 | 58618.85083 | 58571.27675 | 58595.58873
PaddleNLP_gpt_for_sequence_classification_bs8_fp32_DP_N1C1_d2sT | 58949.09969 | 59032.05652 | 59016.48243
PaddleNLP_gpt_for_sequence_classification_bs8_fp16_DP_N1C8 | 668752.57752 | 679106.27208 | 681527.69424
PaddleNLP_gpt_for_sequence_classification_bs8_fp16_DP_N1C8_d2sT | 640749.70464 | 647502.538 | 635732.24232
PaddleNLP_gpt_for_sequence_classification_bs8_fp32_DP_N1C8 | 449825.98232 | 449950.20024 | 449489.38536
PaddleNLP_gpt_for_sequence_classification_bs8_fp32_DP_N1C8_d2sT | 438283.35208 | 438023.05368 | 437547.6752

**Review by JunnYu — APPROVED** [review]

File: tests/test_tipc/configs/gpt_for_sequence_classification/train_infer_python.txt (hunk 1 above)
Comment (FrostML) [reply tests/test_tipc/configs/gpt_for_sequence_classification/train_infer_python.txt]: Done. Thanks.

**Pull request merged**